test/functional/http/get/transactions.js

```diff
@@ -785,8 +786,8 @@ describe('GET /api/transactions', function () {
 
 				return getQueuedTransactionPromise(transaction.id).then(function (result) {
 					node.expect(result).to.have.property('success').to.equal(true);
-					node.expect(result).to.have.property('transaction').that.is.an('object');
-					node.expect(result.transaction.id).to.equal(transaction.id);
+					node.expect(result).to.have.property('transactions').to.be.an('array');
+					node.expect(result.transactions.indexOf(transaction.id)).to.equal(-1);
```

Comment:
Why? That transactions should be there probably

Reply:
Will double check it.

Reply:
fixed